### ISSUE TITLE
ovirt: Should NOT make the "restart", "shutdown", "delete" buttons available for non-root user. Currently, the buttons are enabled, when clicking, then error messages display in UI.

### ISSUE THREAD

**Issue by kenzhaoyihui** [cockpit-project/cockpit#8199]
Description of problem:
Should NOT make the "restart", "shutdown", "delete" buttons available for non-root user. Currently, the buttons are enabled, when clicking, then error messages display in UI.

Version-Release number of selected component (if applicable):
RHEL 7.5
cockpit-ws-154-1.el7.x86_64
cockpit-bridge-154-1.el7.x86_64
cockpit-system-154-1.el7.noarch
cockpit-ovirt-154-1.el7.x86_64

How reproducible:
100%

Steps to Reproduce:
1. Install the pkg cockpit-ovirt on the RHEL7.5
2. Login to the cockpit with non-root user
3. Restart, shutdown, delete the VM

Actual results:
After step3, cockpit display warning messages
"""
 restart vm:    VM REBOOT action failed -- error: failed to connect to the hypervisor, error:authentication failed:access denied by policy
 shutdown vm:   VM SHUT DOWN action failed -- error: failed to connect to the hypervisor, error:authentication failed:access denied by policy
 delete vm:     error: failed to connect to the hypervisor error: authentication failed: access denied by policy
"""

Expected results:
Don't give the available "restart, shutdown, delete" buttons to click.

Additional info:
Also, after step3, the warning icon which always exists. Should reload the browser to disappear the icon.

The picture:
[Cockpit ovirt](https://github.com/kenzhaoyihui/issue_picture/blob/master/cockpitovirt2.png)

**Comment by mareklibra** [comment]
No matter its common behavior, there's no guarantee the `root` can access libvirt in non-read-only mode.
Depending on actual Libvirt configuration, a non-root user might even be granted to access it while root is not.

Recent behavior relies on virsh which either passes or fails while enforcing the read-only mode whenever possible.

To improve the behavior from user perspective, there shall be test whether the user can invoke `virsh` in non-read-only mode (means without `-r` option) invoked right after initialization. If it fails, write-actions will be disabled within the UI (means visible but unclickable).

Within implementation, the test can be composed of two Promises, one handling the timeout and the inner one spawning the virsh, i.e. `virsh list`.

**Comment by mareklibra** [comment]
With #9226 merged, cockpit-machines-ovirt can invoke "write" actions on libvirt.

So scope of this issue is changed:
- VM is managed by oVirt
  - try invoke the command via oVirt API
  - if it fails, invoke via Libvirt after user confirmation
- VM is local, non-managed by oVirt
  - call libvirt command immediately

**Comment by mareklibra** [comment]
Closing as duplicate of #7670
